server/app/views/components/ProgramCardFactory.java

```diff
@@ -239,12 +250,20 @@ public static Comparator<ProgramCardData> programTypeThenLastModifiedThenNameCom
                 getDisplayProgram(cardData).localizedName().getDefault().toLowerCase(Locale.ROOT));
   }
 
+  /** Represents the type of admin that's viewing the program card. */
+  public enum AdminType {
+    CIVIFORM_ADMIN,
+    PROGRAM_ADMIN,
+  }
```

Comment:
(blocking) Instead of create a new enum please switch this to use `auth.Role`

Reply:
Good idea, updated to use `CiviFormProfile` since both views had that immediately available.